analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.Vsix.csproj

```diff
@@ -122,16 +120,20 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Google.Protobuf, Version=3.6.1.0, Culture=neutral, PublicKeyToken=a7d26565bac4d604, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Google.Protobuf.3.6.1\lib\net45\Google.Protobuf.dll</HintPath>
```

Comment:
I like a lot more `PackageReference`. No more relative paths.

Reply:
🎉 It's so much better now